packages/web-react/src/hooks/useCancelEvent.ts

```diff
@@ -1,25 +1,61 @@
-import { useCallback, useEffect, MutableRefObject } from 'react';
+import { useCallback, MutableRefObject } from 'react';
+import { useIsomorphicLayoutEffect } from './useIsomorphicLayoutEffect';
+
+// TODO: Remove `isOpen` and listeners with `handleKeyDown` when Chrome fixes the bug,
```

Comment:
Do you find any bug reports related to this issue? Maybe some links to it will be helpful.

Reply:
reported here:
https://issues.chromium.org/issues/351867704